tutorial/training/gocean/2.10-GameOfLife-mpi/psy.f90

```diff
@@ -0,0 +1,106 @@
+  MODULE psy_time_step_alg_mod
```

Comment:
What's the purpose of this file? I would have thought we just generated this with PSyclone?